doc/whats_new.rst

```diff
@@ -7,6 +7,8 @@ What's new
     Note, we are now using links to highlight new functions and classes.
     Please be sure to follow the examples below like :func:`mne.stats.f_mway_rm`, so the whats_new page will have a link to the function/class documentation.
 
+.. currentmodule:: mne
```

Comment:
Nice little tweak here @agramfort now we can reference things more easily in `whats_new.py` like we do elsewhere in the docs (no need to prepend `mne.`)